file_store.go

```diff
@@ -29,6 +29,9 @@ import (
 // to keep the credentials in plain-text.
 //
 // Reference: https://docs.docker.com/engine/reference/commandline/cli/#docker-cli-configuration-file-configjson-properties
+//
+// Deprecated: This type is deprecated.
+// The same functionality is now provided by oras-go.
 type FileStore struct {
```

Comment:
Besides adding `Deprecated` mark, you should also change the implementation. For example,
```go
type FileStore = credentials.FileStore
```